instaloader/nodeiterator.py

```diff
@@ -2,6 +2,7 @@
 import hashlib
 import json
 import os
+from .exceptions import QueryReturnedBadRequestException
```

Comment:
There is already a `from .exception import` six lines below, where the new import fits better.